packages/desktop-client/src/accounts/accountsSlice.ts

```diff
@@ -233,25 +233,41 @@ export const syncAccounts = createAppAsyncThunk(
       return false;
     }
 
-    const batchSync = !id;
+    const { setAccountsSyncing } = accountsSlice.actions;
+
+    if (id === 'uncategorized') {
+      // Sync no accounts
+      dispatch(setAccountsSyncing({ ids: [] }));
+      return false;
+    }
```

Comment:
I haven't actually looked at the code, but is accountId uncategorized an option? It sounds odd to me

Reply:
The `uncategorized` transaction page uses it (similar to offbudget/on-budget page). It currently does not have a bank sync button there though, but it doesn't hurt to also account for this edge case scenario if we want to be thorough with all the possible combinations of account-ids in URLs.